jcl/src/java.base/share/classes/jdk/internal/foreign/abi/UpcallLinker.java

```diff
@@ -102,7 +102,15 @@ public static MemorySegment make(MethodHandle target, MethodType mt, FunctionDes
 
 	/*[IF JAVA_SPEC_VERSION >= 21]*/
 	/**
-	 * A stub method to be implemented.
+	 * A stub method instended in OpenJDK to support compilation.
+	 *
+	 * Note:
+	 * SharedUtils.arrangeUpcallHelper() (totally featured with the
+	 * signatures intended for OpenJDK) calls this method which
+	 * is specific to the upcall specific code implemented in
+	 * OpenJDK. To work around this case during the compilation,
+	 * this stub method serves as a placeholder to minimize the
+	 * changes in OpenJDK.
```

Comment:
Please correct the rest of this javadoc comment (incomplete `@` elements).

Reply:
Fixed.